src/merge.ts

```diff
@@ -242,8 +255,9 @@ export function mergeWithMapping(inFilesContent: string | string[], destFileCont
                 destUnit.attr.id = unit.attr.id;
                 resetTranslationState(destUnit, xliffVersion, options);
             }
-
-            syncOtherNodes(unit, destUnit, 'source', 'target', 'segment');
+            if (sync) {
```

Comment:
if I understand correctly, it would not harm to remove this condition and always do `syncOtherNodes(..)`? (In case I did not oversee anything, I'd rather remove the condition and `sync` variable to have a little less complexity ;) )

Reply:
I case of `overwriteTargetWithTranslated` I do `syncOtherNodes` with different `ignoreElementsNames`.

Reply:
Yes, but here you ignore more, so there shouldn't any difference if it is run again?

Reply:
ah, now I get your point. You're right! :) 
I have removed it.